config/config_test.go

```diff
@@ -343,6 +343,11 @@ func TestConfigInvariant(t *testing.T) {
 	err = codecs.LoadObjectFromFile(filepath.Join(configsPath, "config-v4.json"), &c4)
 	a.NoError(err)
 	a.Equal(defaultLocalV4, c4)
+
+	c5 := Local{}
```

Comment:
I think that you should following the configuration patten that I've used for the rate limiting : https://github.com/algorand/go-algorand/pull/151

Instead of defining a new configuration version, add it to the recent version and provide upgrade support.

Reply:
I'll only do that if I can rewrite all the loud comments in config/local_defaults.go to describe an actual process we actually do

Reply:
OK I heard you.  Yes the comments are in-your-face and no longer apply.  You now do NOT need to create a new version or bother with migration UNLESS YOU ARE CHANGING THE DEFAULTS FOR AN EXISTING PARAMETER.